packages/react/package.json

```diff
@@ -19,11 +19,11 @@
     "Christian Murphy <christian.murphy.42@gmail.com>"
   ],
   "license": "MIT",
-  "main": "dist/mdx-react.js",
+  "main": "dist/mdx-react.cjs",
```

Comment:
`"type": "module"` implies that `dist/mdx-react.js` is ESM. `dist/mdx-react.cjs` is identical but explicitly CJS. This avoids:
```
FAIL test/index.test.js
  ● Test suite failed to run

    SyntaxError: The requested module '@mdx-js/react' does not provide an export named 'MDXProvider'

      at Runtime.linkAndEvaluateModule (../../node_modules/jest-runtime/build/index.js:669:5)
      at TestScheduler.scheduleTests (../../node_modules/@jest/core/build/TestScheduler.js:333:13)
      at runJest (../../node_modules/@jest/core/build/runJest.js:387:19)
      at _run10000 (../../node_modules/@jest/core/build/cli/index.js:408:7)
      at runCLI (../../node_modules/@jest/core/build/cli/index.js:261:3)